docs/modules/operation/pages/flows/setup.adoc

```diff
@@ -114,88 +114,61 @@ The value used for the lookup corresponds to the following fields from the vario
+| IPFix        | observationDomainId
+| SFlow        | agent_address:sub_agent_id
+| BMP          | bgpId
 |===
 
-== Node Cache Configuration (Optional)
+== Node cache configuration
 
-By default, each flow document - if {page-component-title} knows it - is  enriched with node information.
+By default, each flow document, if {page-component-title} can match the IP address to a node, is enriched with node information.
 To reduce the number of queries to the database, the data is cached.
 
-The following cache properties are available to be set in `$\{OPENNMS_HOME/etc/org.opennms.features.flows.persistence.elastic.cfg`:
+The following cache properties are available to be set in `$OPENNMS_HOME/etc/org.opennms.features.flows.persistence.elastic.cfg`:
 
 [options="header, autowidth"]
 |===
-| Property | Description | Required | Default
-
-| `nodeCache.maximumSize`
-| The maximum size of the cache.
-| `false`
-| `1000`
-
-| `nodeCache.expireAfterWrite`
-| Number of seconds until an entry in the node cache is evicted. Set to 0 to disable eviction.
-| `false`
-| `300`
-
-| `nodeCache.recordStats`
-| Defines if cache statistics are exposed via JMX. Set to `false` to disable statistic recording.
-| `false`
-| `true`
-
+| Property                   | Default | Description
+3+| *Optional*
+| nodeCache.maximumSize      | 1000    | The maximum size of the cache.
+| nodeCache.expireAfterWrite | 300     | Number of seconds until an entry in the node cache is evicted. Set to 0 to disable eviction.
+| nodeCache.recordStats      | true    | Defines if cache statistics are exposed via JMX. Set to `false` to disable statistic recording.
 |===
 
-== Classification Exporter Filter Cache Configuration (Optional)
+== Classification exporter filter cache configuration
 
 A rule in the Classification Engine may define an `exporterFilter`.
 To resolve if the filter criteria match the address of an exporter, a database query is executed.
 A cache can be configured to cache the result to improve performance.
 
-The following cache properties are available to be set in `$\{OPENNMS_HOME/etc/org.opennms.features.flows.classification.cfg`:
+The following cache properties are available to be set in `$OPENNMS_HOME/etc/org.opennms.features.flows.classification.cfg`:
 
 [options="header, autowidth"]
 |===
-| Property | Description | Required | Default
-
-| `cache.classificationFilter.enabled`
-| Enables or disables the cache.
-| `false`
-| `false`
-
-| `cache.classificationFilter.maxSize`
-| The maximum size of the cache.
-| `false`
-| `5000`
-
-| `cache.classificationFilter.expireAfterRead`
-| Number of seconds until an entry in the node cache is evicted. Set to 0 to disable eviction.
-  The timer is reset every time an entry is read.
-| `false`
-| `300`
-
-| `nodeCache.recordStats`
-| Defines if cache statistics are exposed via JMX. Set to `false` to disable statistic recording.
-| `false`
-| `true`
-
+| Property                                   | Default | Description
```

Comment:
Same as above, we can say "The following optional cache properties ..." then remove the "Optional" row from the table.